Docs/svt-hevc_encoder_user_guide.md

```diff
@@ -111,11 +111,13 @@ The list below includes the build tools necessary for the encoder application an
   - GCC 5.4.0
   - CMake 3.5.1
   - YASM Assembler version 1.2.0 or later
-  
+
```

Comment:
Adding new empty lines doesn't keep sync with above windows instruction. 
Either remove new lines or make them the same.

Reply:
Seems to look correct when I preview the doc.  Could you describe more?

Reply:
There are new lines added in your patch.
While the same block for windows doesn't show empty lines in between.
You can view this md(not diff) this way: "files changed" -> "..." -> "view file"